coalib/output/ConsoleInteraction.py

```diff
@@ -152,14 +153,36 @@ def acquire_actions_and_apply(console_printer,
             metadata_list.append(metadata)
 
         # User can always choose no action which is guaranteed to succeed
-        if not ask_for_action_and_apply(console_printer,
-                                        section,
-                                        metadata_list,
-                                        action_dict,
-                                        failed_actions,
-                                        result,
-                                        file_diff_dict,
-                                        file_dict):
+        if section.name == 'cli':
+            if not ask_for_action_and_apply(console_printer,
+                                            section,
+                                            metadata_list,
+                                            action_dict,
+                                            failed_actions,
+                                            result,
+                                            file_diff_dict,
+                                            file_dict):
+                break
+            question = format_lines(
+                'Reprompt available actions (Y/N)? ', symbol='!')
+            input_q = input(question)
+            input_q = str(input_q).lower()
+            if input_q in ['yes', 'yea', 'y']:
```

Comment:
I dont like the hardcoding. Isnt there a "constants" file somewhere, or just define a a global constant list if not? But therr should be one.